src/time-slot-bar-week/time-slot-bar-week-days.tsx

```diff
@@ -152,27 +156,31 @@ export const TimeSlotBarWeekDays = ({
     const generateStyleProps = (day: Dayjs) => {
```

Comment:
to also include the greyed out styling for non-current months. it would be something like

```
 if (calendarDate.month() !== day.month()) {
    labelType = "unavailable";
}
```

`CellWeekText` should also apply this color

Reply:
i think we should leave the greyed out style exclusive to disabled dates, having non-current month days greyed out can be confusing when they're actually available

Reply:
I see, the current TimeSlotWeekView does indeed not grey out non-current month days. I may have confused it with the calendar